browser/ui/toolbar/brave_app_menu_model.h

```diff
@@ -46,17 +46,27 @@ class BraveAppMenuModel : public AppMenuModel {
   absl::optional<size_t> GetProperItemIndex(std::vector<int> commands_to_check,
                                             bool insert_next) const;
 
-  void InsertBraveMenuItems();
-  void InsertAlternateProfileItems();
-  size_t GetIndexOfBraveRewardsItem() const;
-  size_t GetLastIndexOfSecondSection() const;
-  size_t GetIndexOfBraveSyncItem() const;
-#if BUILDFLAG(ENABLE_BRAVE_VPN)
-  size_t GetIndexOfBraveVPNItem() const;
-#endif
-#if defined(TOOLKIT_VIEWS)
-  size_t GetIndexOfBraveSidebarItem() const;
-#endif
+  // Tab & windows.
+  void BuildFirstSection();
+
+  // Leo, Wallet, VPN and IPFS.
+  void BuildSecondSection();
+  size_t GetNextIndexOfSecondSection() const;
+
+  // History, bookmarks, downloads and extensions.
+  void BuildThirdSection();
```

Comment:
Very optional :laughing:
```suggestion
  void BuildBrowserSection();
```